doc/authentication-backends/PKI-authentication-module.md

```diff
@@ -0,0 +1,24 @@
+## Overview
+
+PKI authentication backend is compatible only with SASL mechanisms, that are based on client certificates.
```

Comment:
The PKI authentication backend is compatible only with SASL mechanisms, that are based on client certificates? 
A PKI authentication backend is compatible only with SASL mechanisms, that are based on client certificates?